python/python/glide/async_commands/cluster_commands.py

```diff
@@ -63,10 +73,16 @@ async def info(
             the queried node and value containing the information regarding the requested sections.
         """
         args = [section.value for section in sections] if sections else []
+        result_bytes = await self._execute_command(RequestType.Info, args, route)
+        result_str = None
+        if isinstance(result_bytes, bytes):
```

Comment:
why don't you put this logic into `convert_byte_to_string_dict` or make another function that does this?  This will probably need to be used throughout the file.

Reply:
As of now, we only have one such case, if we find more, we'll move it to a function